### PR TITLE
Document the feature `android.allow_custom_android_id`

### PR DESCRIPTION
**Pull request by adglkh**

### CODE REVIEW

File: ref/ams-configuration.tmpl.md [hunk 1]
```diff
@@ -82,3 +82,18 @@ Once set, this feature flag will be considered by all newly launched containers.
 The Android development settings (which include an ADB connection) are enabled by default. Some applications require these settings to be disabled, which you can do with the `disable_development_settings` feature flag.
 
 Once set, this feature flag will be considered by all newly launched containers.
+
+#### Custom Android ID
+
+*since 1.18.0*
+
+To enable the Android container to use a custom Android ID, add the feature `android.allow_custom_android_id` upon application creation. An system app can influence the Android ID of one specific app during the Android runtime by setting the system property in the format of:
```
Comment (keirthana) [inline ref/ams-configuration.tmpl.md:90]:
```suggestion
To enable the Android container to use a custom Android ID, add the feature `android.allow_custom_android_id` upon application creation. A system app can influence the Android ID of a specific app during the Android runtime by setting the system property in the format of:
```

Comment (keirthana) [reply ref/ams-configuration.tmpl.md:90]: I am not really sure but should it read:

> To enable the Android container to use a custom Android ID, add the feature _flag_ `android.allow_custom_android_id` upon application creation.

**Review by keirthana — CHANGES_REQUESTED** [review]
Minor comments from me, if these are addressed, I am good.

**Review by keirthana — APPROVED** [review]
LGTM!

File: ref/ams-configuration.tmpl.md [hunk 2]
```diff
@@ -82,3 +82,18 @@ Once set, this feature flag will be considered by all newly launched containers.
...
+#### Custom Android ID
+
+*since 1.18.0*
+
+To enable the Android container to use a custom Android ID, add the feature flag `android.allow_custom_android_id` upon application creation. A system app can influence the Android ID of a specific app during the Android runtime by setting the system property in the format of:
+  ```
+  `anbox.custom_android_id.<index>=<package_name>:<android_id>`
+  ```
+
+  The `<index>` is a number in the range from 0 to 126.
```
Comment (morphis) [inline ref/ams-configuration.tmpl.md:95]: We should explain the index a bit further so it's clear why it exists.

**Review by morphis — CHANGES_REQUESTED** [review]

**Review by morphis — APPROVED** [review]
LGTM

**Pull request merged**